package.nls.hindi.json

```
@@ -0,0 +1,31 @@
+{
+    "python.command.python.sortImports.title": "आयात क्रमबद्ध करें",
+    "python.command.python.startREPL.title": "REPL शुरू करें",
+    "python.command.python.createTerminal.title": "टर्मिनल बनाएं",
+    "python.command.python.buildWorkspaceSymbols.title": "वर्कस्पेस सिंबल बनाएं",
+    "python.command.python.runtests.title": "सभी टेस्ट चलाएं",
+    "python.command.python.debugtests.title": "सभी टेस्ट डीबग करें",
+    "python.command.python.execInTerminal.title": "टर्मिनल में पायथन फाइल चलाएं",
+    "python.command.python.setInterpreter.title": "इंटरप्रेटर चुनें",
+    "python.command.python.refactorExtractVariable.title": "Variable निकालें",
+    "python.command.python.refactorExtractMethod.title": "Method निकालें",
```

```suggestion
    "python.command.python.refactorExtractVariable.title": "वेरिएबल निकालें",
    "python.command.python.refactorExtractMethod.title": "मेथड निकालें",
```
Non blocking. I feel these should be in the same script, since test/terminal are in that script.